src/converter/internal/convertercontroller.cpp

```diff
@@ -181,7 +181,8 @@ mu::Ret ConverterController::convertPageByPage(INotationWriterPtr writer, INotat
     TRACEFUNC;
 
     for (size_t i = 0; i < notation->elements()->pages().size(); i++) {
-        const QString filePath = io::path_t(io::dirpath(out) + "/" + io::basename(out) + "-%1." + io::suffix(out)).toQString().arg(i + 1);
+        const QString filePath = io::path_t(io::dirpath(out) + "/" + io::completeBasename(out) + "-%1." \
```

Comment:
You don't need to add a backslash in C++ (that's why there are semicolons :) )

Reply:
I just added a '\n'(entry) before, and it still failed. And I search on the Internet for what I should do when a code statement is too long, and that's the reason why I add '\' 😂